test/functional/core/job_spec.lua

```diff
@@ -273,7 +268,7 @@ describe('jobs', function()
   end)
 
   it('disallows jobstop twice on the same job', function()
-    nvim('command', "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
+    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
```

Comment:
Removing these "find" workarounds is likely to fail. We already tried cat.exe for these cases and there were inconsistencies on windows/appveyor.

Reply:
There is only one real failure/difference it seems (see fb43902).